src/App.css

```diff
@@ -1,4 +1,320 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700&family=Roboto+Slab:wght@300;400;500;600;700&display=swap');
+
+:root {
+  --primary-color: #0290ff;
+  --pale-blue: #4386bf;
+}
+
 body {
   margin: 0;
   overflow-x: hidden;
+  font-family: 'Montserrat', sans-serif;
+  color: #121212;
+  background-color: #fafafa;
+}
+
+header {
```

Comment:
- [ ] Please make sure that you added all elements in design. For example, in the navbar of your app there is no a person avatar like in the given original design. Take a look at the screenshots below to see the difference 👇 
- Original design 👇 
![originale](https://user-images.githubusercontent.com/67087664/175767968-73f0c94c-1e67-4559-9e29-ec6b566230b8.png)
- Your design 👇 
![Screenshot (284)_LI](https://user-images.githubusercontent.com/67087664/175767990-f24c4a0b-e01e-46d6-8b8e-0514769ee4ed.jpg)